app/uk/gov/hmrc/plasticpackagingtax/registration/connectors/EmailVerificationConnector.scala

```diff
@@ -119,7 +119,22 @@ class EmailVerificationConnector @Inject() (
       }
   }
 
+  def getTestOnlyPasscode()(implicit hc: HeaderCarrier): Future[Either[ServiceError, String]] =
+    httpClient.GET[HttpResponse](appConfig.getTestOnlyPasscodeUrl)
+      .map {
+        case response @ HttpResponse(OK, _, _) =>
+          Right(response.body)
+        case response =>
+          Left(
+            DownstreamServiceError(
+              s"Unable to find test only passcodes :, status: ${response.status}, error: ${response.body}",
+              FailedToFetchTestOnlyPasscode("Failed to get test only passcodes")
+            )
+          )
+      }
+
```

Comment:
I think we should extract this into a test only connector so that this is not confused with proper production code.

Reply:
Moved it to testOnly package